modules/vault-cluster/main.tf

```diff
@@ -88,7 +198,7 @@ resource "aws_security_group" "lc_security_group" {
     create_before_destroy = true
   }
 
-  tags = "${merge(map("Name", var.cluster_name), var.security_group_tags)}"
+  tags = "${merge(var.security_group_tags, map("Name", var.cluster_name))}"
```

Comment:
Why reverse the order of the tags?

Reply:
I don't want var.security_group_tags to inadvertently change the value of the Name tag.

Reply:
Hm, I'm tempted to say that's a feature, not a bug, as it allows the user full control over the `Name` tag in case the default isn't what they want...